test/exchange-connector/exchangeTools.test.ts

```diff
@@ -184,3 +186,43 @@ describe("Test groupByCategory method", () => {
     });
   });
 });
+
+describe("removeRamlLinks", () => {
+  const REST_APIS: RestApi[] = [
+    {
+      id: "8888888/test-api/1.0.0",
+      name: "Test API",
+      groupId: "8888888",
+      assetId: "test-api",
+      fatRaml: {
+        classifier: "rest-api",
+        sha1: "sha1",
+        md5: "md5",
+        externalLink: "https://somewhere/fatraml.zip",
+        packaging: "zip",
+        createdDate: "today",
+        mainFile: "api.raml"
+      }
+    },
+    {
+      id: "8888888/test-api2/1.0.0",
+      name: "Test API",
+      groupId: "8888888",
+      assetId: "test-api",
+      fatRaml: {
+        classifier: "rest-api",
+        sha1: "sha1",
+        md5: "md5",
+        externalLink: "https://somewhere/fatraml2.zip",
+        packaging: "zip",
+        createdDate: "today",
+        mainFile: "api.raml"
+      }
+    }
+  ];
+
+  it("removes fat raml external links for all the apis", () => {
+    const apis = _.cloneDeep(REST_APIS);
+    expect(removeRamlLinks(apis)).to.not.contain("externalLink");
+  });
```

Comment:
Can we pls add tests for arrays that are 
1. empty
2. null
3. undefined
4. containing objects without external Link

Reply:
done